src/mmrelay/__main__.py

```
@@ -0,0 +1,25 @@
+import sys
+
+if __name__ == "__main__":
+    try:
+        from mmrelay.cli import main
+
+        sys.exit(main())
+    except ImportError as e:
+        print(f"Error importing MMRelay CLI: {e}", file=sys.stderr)
+        print("Please ensure MMRelay is properly installed.", file=sys.stderr)
+        sys.exit(1)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)